docs/installation.md

```diff
@@ -1,9 +1,16 @@
 # Installing Ruff
 
-Ruff is available as [`ruff`](https://pypi.org/project/ruff/) on PyPI:
+Ruff is available as [`ruff`](https://pypi.org/project/ruff/) on PyPI. We recommend installing Ruff with [uv](https://docs.astral.sh/uv/):
 
 ```console
+$ # With uv.
```

Comment:
We should only use a leading `$` in the documentation if we add the copy/paste fix javascript from uv.

Reply:
Okay thanks. We already use this in a few places, I think. I'll figure out what to do!